### ISSUE TITLE
oxTrust still broken in gluu 2.3.3-1

### ISSUE THREAD

**Issue by rngadam** [GluuFederation/community-edition-setup#59]
This issue is marked as closed: https://github.com/GluuFederation/oxTrust/issues/74

However, after installation of gluu 2.3.3-1 (latest on Ubuntu 14.04 from official packages) oxTrust still breaks:

```
INFO   | jvm 1    | 2015/09/07 04:17:32 | java.lang.ArrayIndexOutOfBoundsException: 1
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.gluu.oxtrust.ldap.service.StatusCheckerTimer.setCertificateExpiryAttributes(StatusCheckerTimer.jav
a:165)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.gluu.oxtrust.ldap.service.StatusCheckerTimer.process(StatusCheckerTimer.java:126)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.gluu.oxtrust.ldap.service.StatusCheckerTimer.scheduleStatusChecking(StatusCheckerTimer.java:90)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at sun.reflect.GeneratedMethodAccessor208.invoke(Unknown Source)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at java.lang.reflect.Method.invoke(Method.java:606)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.util.Reflections.invoke(Reflections.java:22)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.intercept.RootInvocationContext.proceed(RootInvocationContext.java:32)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.intercept.SeamInvocationContext.proceed(SeamInvocationContext.java:56)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.transaction.RollbackInterceptor.aroundInvoke(RollbackInterceptor.java:28)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.intercept.SeamInvocationContext.proceed(SeamInvocationContext.java:68)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.core.BijectionInterceptor.aroundInvoke(BijectionInterceptor.java:79)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.intercept.SeamInvocationContext.proceed(SeamInvocationContext.java:68)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.core.MethodContextInterceptor.aroundInvoke(MethodContextInterceptor.java:44)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.intercept.SeamInvocationContext.proceed(SeamInvocationContext.java:68)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.async.AsynchronousInterceptor.aroundInvoke(AsynchronousInterceptor.java:52)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.intercept.SeamInvocationContext.proceed(SeamInvocationContext.java:68)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.intercept.RootInterceptor.invoke(RootInterceptor.java:107)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.intercept.JavaBeanInterceptor.interceptInvocation(JavaBeanInterceptor.java:196)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.jboss.seam.intercept.JavaBeanInterceptor.invoke(JavaBeanInterceptor.java:114)
INFO   | jvm 1    | 2015/09/07 04:17:32 |       at org.gluu.oxtrust.ldap.service.StatusCheckerTimer_$$_javassist_seam_7.scheduleStatusChecking(StatusCheckerTimer_$$_javassist_seam_7.java)
```

Was the change unreleased?

**Comment by aliaksander-samuseu** [comment]
This issue doesn't seem to affect any functionality, but it does clutter logs, and produces false alarms from users when resolving issues. Is it possible to get rid of it somethow?

**Comment by willow9886** [comment]
It appears this has been fixed and will be released in version 2.4.0.

**Comment by yurem** [comment]
It fixed in CE 2.3.4 Also this fix it will be part of CE 2.4.0
We can reccomend to use latest oxTrust from 2.3 branch, or install CE 2.3.4
This is latest oxTrust CE 2.3.4 war: http://ox.gluu.org/maven/org/xdi/oxtrust-server/2.3.4.Final/oxtrust-server-2.3.4.Final.war
